docs/commcare-cloud/commands/index.md

```diff
@@ -161,7 +161,7 @@ multiple servers if there is more than one in the group. For
 example: webworkers:0 will pick the first webworker. May also be
```

Comment:
Should `webworkers:0` to be changed to `webworkers[0]`? (Other parts of this paragraph need to be rewritten too if yes.) Same below on line 189, 218, etc. There's another similar paragraph at https://github.com/dimagi/commcare-cloud/blob/6cb734133525f96cf8bdf0da3a680ab686c91a3a/src/commcare_cloud/commands/inventory_lookup/inventory_lookup.py#L27-L29

Reply:
https://github.com/dimagi/commcare-cloud/pull/3987